jiant/models.py

```diff
@@ -571,23 +578,26 @@ def build_image_sent_module(task, d_inp, params):
     return pooler
 
 
-def build_single_sentence_module(task, d_inp: int, use_bert: bool, params: Params):
+def build_single_sentence_module(task, d_inp: int, use_pytorch_transformers: bool, params: Params):
     """ Build a single sentence classifier
 
     args:
         - task (Task): task object, used to get the number of output classes
         - d_inp (int): input dimension to the module, needed for optional linear projection
-        - use_bert (bool): if using BERT, skip projection before pooling.
+        - use_pytorch_transformers (bool): if using BERT/XLNet, skip projection before pooling.
```

Comment:
Perhaps drop the pytorch in the variable name since jiant is all in pytorch anyways

Reply:
I'd rather keep it clear that this is about the `pytorch-transformers` package, not about the general idea of a transformer. For example, GPT doesn't currently use this property.